### DISCUSSION TITLE
Deploy Manager without search node

### DISCUSSION THREAD

**Discussion by mikos2233** [Security-Onion-Solutions/securityonion#2242] [Unsupported Versions] [answered]
Hello,
I want to deploy a Manager node without the search node to lower the hardware requirements (and use a VM for manager node) as written here:
<img width="815" alt="Zrzut ekranu 2020-12-7 o 20 57 08" src="https://user-images.githubusercontent.com/75631635/101398843-fcc32d00-38ce-11eb-8c22-f2fcfc7bff18.png">

I'm a litte bit confused during the installation process.
Should I choose Manager or Manager+Search?

**Comment by TOoSmOotH** [comment, marked as answer]
Manager role is just the manager. There is no search on it.

**Reply by mikos2233** [reply]
I'm sorry I posted a wrong picture - I meant this:
<img width="815" alt="Zrzut ekranu 2020-12-7 o 20 57 08" src="https://user-images.githubusercontent.com/75631635/101398762-df8e5e80-38ce-11eb-96f2-1e1d9ece5ef4.png">


So if I'm understanding correctly, to use the "Manager node with separate search nodes (low hardware requirement)" option I should choose "Manager" option during install?

**Reply by dougburks** [reply]
Yes!
